src/components/TextArea/TextArea.driver.ts

```diff
@@ -2,6 +2,7 @@ import {
   BaseUniDriver,
   baseUniDriverFactory,
 } from 'wix-ui-test-utils/base-driver';
+import { Simulate } from 'react-dom/test-utils';
```

Comment:
I'm assuming what I suggested didn't work?

This isn't fit for unidriver's though.
The unidriver should be environment agnostic, so we shouldn't use tools specific for react-dom/jsdom.

I guess we'll have to move this logic to the test itself.
So you'll need to expose the `textarea` function and move the simulation to the test itself.

Reply:
OK, I thought this might be acceptable since I saw other drivers use Simulate (the Checkbox driver, for example).

Reply:
You're right.
We really need to fix that.